util/logutil/slow_query_logger.go

```diff
@@ -32,6 +32,9 @@ func newSlowQueryLogger(cfg *LogConfig) (*zap.Logger, *log.ZapProperties, error)
 	// copy the global log config to slow log config
 	// if the filename of slow log config is empty, slow log will behave the same as global log.
 	sqConfig := cfg.Config
+	// level of the global log config doesn't affect the slow query logger which determines whether to
+	// log by execution duration.
+	sqConfig.Level = LogConfig{}.Level
```

Comment:
`LogConfig{}.Level` set slowlogger to `info` level implicitly. Since `executor/adapter.go` logs slowlog with `warn()` method, this implicit setting could hide necessary information from developers.
```suggestion
	sqConfig.Level = zap.InfoLevel.String()
```

Reply:
Yes, I fixed the bug just like you at begin. But I talked about it with @youjiali1995 and We two think that It‘s best not to expose zap.InfoLevel here.
Anyway, I think the two fixed methods all can work.

Reply:
I would prefer to explicitly indicate that the default log level is INFO, and slow log will always use WARN to log so that it is always printed out.

Otherwise as a reader I will have no idea about how slow log will be outputted. What do you think? @youjiali1995

Reply:
Using the default value indicates slow logger doesn't care about log level. I would like to delete the `Level` field of the config if I could. 

There is a test to cover the constraint that the default log level is less than warn level.